### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by leonardocelis112** [langflow-ai/langflow#3354]
### Bug Description

I followed the steps on the getting started.

I installed with both pip and pipx and in every installation i got the follwoing error.

ImportError: cannot import name 'ToolsRenderer' from 'langchain_core.tools' (/Users/leonardocelis/.pyenv/versions/3.11.3/lib/python3.11/site-packages/langchain_core/tools/__init__.py)

It seems is not able to find the ToolsRenderer class.

Maybe a langchain version issue.

Thanks in advance.

### Reproduction

1. install a fresh environment with the desired python version
2. install running python -m pip install langflow -U
3. try to run with python -m langflow run
4. the error mentioned appears

### Expected behavior

Expect the langflow server to run

### Operating System

Mac OS

### Langflow Version

1.0.15

### Python Version

3.11

### Screenshot

<img width="1021" alt="image" src="https://github.com/user-attachments/assets/963d0127-6834-4466-a1c6-7c543ae5e276">

**Comment by evbz22** [comment]
Still errors out even after trying

```
pip install langchain==0.6.7
ERROR: Could not find a version that satisfies the requirement langchain==0.6.7


ERROR: No matching distribution found for langchain==0.6.7
```

**Comment by leonardocelis112** [comment]
I'm having the same error that @evbz22 has.

**Comment by evbz22** [comment]
Found a resolution to this issue.
Langchain seems to have released langchain: release 0.2.14 #25416 to address this issue.
https://github.com/langchain-ai/langchain/pull/25416

Please check https://github.com/langchain-ai/langchain/issues/25413
It fixed my issue.
